app/reducers/index.js

```diff
@@ -229,3 +183,93 @@ export const surveySubmissionSchema = new schema.Entity('surveySubmissions', {
   user: userSchema,
 });
 export const tagSchema = new schema.Entity('tags', {}, { idAttribute: 'tag' });
+
+const followersSchemaGenerator = (
+  title,
+  types: {
+    fetch?: EntityReducerTypes,
+    mutate?: EntityReducerTypes,
+    delete?: EntityReducerTypes,
+  } = {}
+) => {
+  const key = `followers${title}`;
+  return [
+    new schema.Entity(key, {
+      follower: userSchema,
+    }),
+    createEntityReducer({
+      key,
+      types,
+    }),
```

Comment:
How does mutating the follower states work now? I'm not familiar with the default handling in `createEntityReducer` :upside_down_face:.

Reply:
It woks as expected (normal CRUD) as long as the actions define the correct schema.